src/chromium/adding-third-party-crates/depending-on-a-crate.md

```diff
@@ -0,0 +1,15 @@
+# Depending on a crate
+
+Once you've added a third-party crate and generated build rules,
+depending on a crate is simple. Find your `rust_static_library` target,
+and add a `dep` on the `:lib` target within your crate.
+
+For instance,
+
+```gn
+rust_static_library("my_rust_lib") {
+  crate_root = "lib.rs"
+  sources = [ "lib.rs" ]
+  deps = [ "//third_party/rust/example_rust_crate/v1:lib" ]
+}
```

Comment:
optional suggestion: Would it be easier for attendees/readers to start with `rust_executable` (I think that this way they could more easily start to actually run code from the newly added third-party crate).

Reply:
OTOH, I see that you *do* mention `rust_executable` in the exercises section.  So probably okay to just resolve my comment here.

Reply:
It's a good point. I've made this more explicitly an optional exercise here.